src/renderers/shared/stack/reconciler/ReactOwner.js

```diff
@@ -7,12 +7,28 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactOwner
+ * @flow
  */
 
 'use strict';
 
 var invariant = require('invariant');
 
+import type { ReactInstance } from 'ReactInstanceType';
+
+/**
+ * @param {?object} object
+ * @return {boolean} True if `object` is a valid owner.
+ * @final
+ */
+function isValidOwner(object: any): bool {
```

Comment:
The only call site of this function gives a variable of type ReactInstance, so if we follow the type system, this function is always going to return true and the invariant is dead code.

The only reason this function exists is that the entire code base is not respecting the type system. So adding a type mixed or any isn't going to make any difference :)

If you insist, I can change it to mixed, I don't really mind